pages/avoapi/authentication.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: 'guide'
+title: 'Authentication'
+abstract: 'Authentication for Avo API'
+---
+
+import Link from '../../components/Link';
+
+> We are currently testing the Avo Public API and it's not yet in general availability. Reach out to us for access.
+
+Avo API uses `Basic authentication` to authenticate requests with avo service accounts.
+
+## Authenticating with Avo API
+
+Avo API accepts service account token as an authentication method.
+
+Authenticate with the API:
+
+1. Create Service Account
+
+   Start by <Link passHref href="/avoapi/authentication#creating-service-accounts"><a>creating a Service Account</a></Link> and store the secret somewhere secure.
+
+2. Authorization Header
+
+   The Avo API expects the authorization header to be set with BASIC auth.
+
+   - Base64 encoded `name:secret` from the service account.
+
+   ```json
+   ("authorization": "Basic <token>")
+   ```
+
+3. Send a request with the header
+
+   ```sh
+   curl -H "authorization: Basic <token>" -X GET https://api.avo.app/workspaces/<WorkspaceId>/branches/<BranchId>/export/v1
+   ```
+
+## Creating Service Accounts
+
+1. Navigate to your <Link passHref href="https://www.avo.app/schemas/default?settings=service-accounts"><a>Service Account Workspace Settings</a></Link> or open the settings cogwheel in the Avo sidebar, select `Workspace Settings` and then open the `Service Accounts` tab.
+
+   > If you don't see the Service Account tab you might not have access yet. Reach out to us.
+
+   <center>
+     <img src={require('../../images/avoapi/service-accounts.png')} />
+   </center>
+
+2. Enter a descriptive name for where you'll be using this service account and press the `Generate Secret` button.
+
+   > Only workspace members with Admin access can create service accounts.
+
+3. Once Avo has generated and securely stored your secret, you get one chance to view it. Copy the secret key and save it securely in your internal systems.
+
+   > Avo postfixes the name with `_sa_[randomId]` so make sure to copy the final name
+
+   <center>
+     <img
+       src={require('../../images/avoapi/create-service-account.png')}
+       width={411}
+     />
+   </center>
+
+   > You will never see the secret key again, so make sure to copy it.
+
+## Deleting a Service Account
+
+If for some reason you need to delete your service account, you can do so from the service account management page.
+
+1. Navigate to your <Link passHref href="https://www.avo.app/schemas/default?settings=service-accounts"><a>Service Account Workspace Settings</a></Link> or open the settings cogwheel in the Avo sidebar, select `Workspace Settings` and then open the `Service Accounts` tab.
+
+2. Press the context menu to the right of the service account you want to remove and select `Delete Service Account`.
+
+3. Press `Delete` again in the confirm window that pops up.
+
+   > This action cannot be undone. Once deleted, there is no way to recover the service account again.
```

Comment:
```suggestion
   > This action cannot be undone. Once deleted, there is no way to recover the service account again. As soon as a service account has been deleted it will stop working immediately.
```

Reply:
Applied in 7320277